web/sites/default/config/block.block.exposedformevents_indexevents_search_index.yml

```diff
@@ -24,6 +24,6 @@ settings:
 visibility:
   request_path:
     id: request_path
-    pages: "/events\r\n/events/past\r\n/events/search\r\n/events/search/*"
+    pages: "/events\r\n/events/past\r\n/events/search\r\n/events/search/*\r\n/events/*"
```

Comment:
Does /events/* supersede /events/past and /events/search?

Reply:
Ah... yes it does. I added the more global path and should have removed the specific ones. I'll fix that now.

Reply:
Fixed... but now I need your review again. 😆